FirefoxPrivateNetworkVPN/Utilities/TunnelConfigurationBuilder.swift

```diff
@@ -26,8 +26,11 @@ struct TunnelConfigurationBuilder {
 
         // peers
         var peerConfigurations: [PeerConfiguration] = []
+        var serverCity = city
 
-        if let server = city.fastestServer,
+        serverCity.setServerSelection()
```

Comment:
if you change `setServerSelection` to something like `setServer` that also returns the server you can do it all on line 33